zerver/lib/actions.py

```diff
@@ -5183,11 +5186,10 @@ def do_get_user_invites(user_profile: UserProfile) -> List[Dict[str, Any]]:
     days_to_activate = settings.INVITATION_LINK_VALIDITY_DAYS
     active_value = confirmation_settings.STATUS_ACTIVE
     revoked_value = confirmation_settings.STATUS_REVOKED
-    
+
     lowest_datetime = timezone_now() - datetime.timedelta(days=days_to_activate)
-    prereg_users = PreregistrationUser.objects.exclude(status=active_value).filter(
-        invited_at__gte=lowest_datetime,
-        referred_by__realm=user_profile.realm)
+    prereg_users = PreregistrationUser.objects.exclude(status=active_value).exclude(status=revoked_value)\
+        .filter(invited_at__gte=lowest_datetime, referred_by__realm=user_profile.realm)
```

Comment:
Should be cleaner to do `exclude(status__in=[active_value, revoked_value])` rather than chanining `.exclude(...).exlucde(...)`.